lib/consensus/isaac.go

```diff
@@ -176,7 +177,7 @@ func (is *ISAAC) GetSyncInfo() (uint64, []string, error) {
 	is.log.Debug("begin ISAAC.GetSyncInfo", "is.nodesHeight", is.nodesHeight)
 	threshold := is.policy.Threshold()
 	if len(is.nodesHeight) < threshold {
-		return 1, []string{}, errors.New("not enought ballots for sync")
+		return 1, []string{}, errors.New(fmt.Sprintf("could not find enough nodes (threshold=%d) above", threshold))
```

Comment:
"above" has to be followed with something. So "above threshold" would be correct, even if it sounds a bit repetitive.